examples/simpleFetch/components.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {connect} from 'react-redux';
+
+import {STATUS} from './dsm';
+
+const Main = ({messages, error}) => {
```

Comment:
Towards improving your PR based on what you mentioned.
Status constants are imported, so we can use them to compare!

```js
const Main = ({ messages, status }) => {
    if (status === STATUS.ERROR) {
        return <div>Error!</div>;
    }

    if (messages) {
        return <List messages={messages}/>;
    }

    return <div>Loading...</div>;
};
```